virttest/qemu_vm.py

```diff
@@ -999,7 +999,7 @@ def set_value(opt_string, key, fallback=None):
             ip_ver = optget("listening_addr")
             if ip_ver:
                 host_ip = utils_net.get_host_ip_address(self.params, ip_ver)
-                spice_opts.append("addr=%s" % host_ip)
```

Comment:
@xutian no this line is redundant now because `spice_addr` in `spice_options` is defined. So spice address will be added by  #L1005 (set_value("addr=%s", "spice_addr")). Otherwise `addr=`  would be placed two times as qemu-kvm -spice parameter